CodeGeneration/Sources/SyntaxSupport/DeclNodes.swift

```diff
@@ -1824,43 +1836,51 @@ public let DECL_NODES: [Node] = [
         name: "Attributes",
         kind: .collection(kind: "AttributeList", collectionElementName: "Attribute"),
         nameForDiagnostics: "attributes",
+        description: "Attributes that are attached to the struct declaration.",
         isOptional: true
       ),
       Child(
         name: "Modifiers",
         kind: .collection(kind: "ModifierList", collectionElementName: "Modifier"),
         nameForDiagnostics: "modifiers",
+        description: "Modifiers that are attached to the struct declaration.",
         isOptional: true
       ),
       Child(
         name: "StructKeyword",
-        kind: .token(choices: [.keyword(text: "struct")])
+        kind: .token(choices: [.keyword(text: "struct")]),
+        description: "The struct declaration keyword."
       ),
       Child(
         name: "Identifier",
-        kind: .token(choices: [.token(tokenKind: "IdentifierToken")])
+        kind: .token(choices: [.token(tokenKind: "IdentifierToken")]),
+        description: "The struct declaration identifier. If the identifier matches a reserved keyword it must be enclosed in backticks."
       ),
       Child(
         name: "GenericParameterClause",
         kind: .node(kind: "GenericParameterClause"),
         nameForDiagnostics: "generic parameter clause",
+        description: "The generic parameters of the struct declaration.",
         isOptional: true
       ),
       Child(
         name: "InheritanceClause",
         kind: .node(kind: "TypeInheritanceClause"),
         nameForDiagnostics: "type inheritance clause",
+        description: "The struct declaration inheritance clause. It may contain one or more type identifiers.",
```

Comment:
I took cues from[ this section](https://github.com/apple/swift-book/blob/main/TSPL.docc/ReferenceManual/Types.md#type-inheritance-clause) of TSPL here. 

Is it necessary to call out that a struct can only conform to one or more protocols and can't actually inherit from another type?

Reply:
I don’t think we should put any restrictions on here saying that the inherited types can only be protocols because that can’t be ensured at parse time (the name resolution happens in the name lookup phase after type checking). As such, you could have a perfectly valid syntax tree where a struct inherits from multiple other structs. 

Mentioning protocols in more of an example-style to aid understanding would be a good option IMO.

Reply:
Got it, thanks for clarifying! Added some extra illustrative examples instead of discussing restrictions.